qa-tests-backend/src/test/groovy/ImageScanningTest.groovy

```diff
@@ -520,7 +520,7 @@ class ImageScanningTest extends BaseSpecification {
         "Scan image"
         String image = IMAGES_FOR_ERROR_TESTS[scanner.name()][testAspect]
         assert image
-        Services.scanImage(image)
+        ImageService.scanImageNoRetry(image, false)
```

Comment:
Why no retry?

Reply:
In this test we verify and expect an error, so retry has no purpose.

Reply:
Sorry there actually were `scanImageNoRetry` that should have been `scanImage`, just not this one.